rskj-core/src/test/java/co/rsk/peg/utils/BridgeEventLoggerTest.java

```diff
@@ -36,7 +36,7 @@ void testLogLockBtc() {
     }
 
     @Test
-    void testLogPeginBtc() {
+    void logPeginBtc() {
```

Comment:
```suggestion
    void assertLogPeginBtc() {
```
wdyt? Not sure if it adds a lot of value, but sounds more descriptive to me